### PR TITLE
the private key will be destroyed when the current wallet has changed

### LINKED ISSUE #222: ProtectedPrivateKeyDisplay should be destroyed on wallet change
To reproduce:

1. Reveal a private key in `Accounts` page
2. Change Wallet before the 10 seconds countdown is done
3. The private key is still revealed but should be hidden.

### PR DESCRIPTION
**Pull request by David0817**
fix #222

### CODE REVIEW

**Comment by vincent-lee90** [comment]
the destroyed will not be called when the wallet change.
because it's the update of models instead of re-render component.
that adding a destroyed method to destroy the intervals is right.

**Comment by evias** [comment]
Address should be on top of public key

**Comment by David0817** [comment]
> Address should be on top of public key + addresses should be in pretty format.
This problem was fixed in PR (#255 ）

**Comment by vincent-lee90** [comment]
rebased pr#177

File: src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts [hunk 1]
```diff
@@ -90,35 +90,25 @@ export class ProtectedPrivateKeyDisplayTs extends Vue {
   }
   /// end-region computed properties getter/setter
 
+  public init(){
```
Comment (evias) [inline src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts:93]: rename to `reset()`

File: src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts [hunk 2]
```diff
@@ -138,4 +128,15 @@ export class ProtectedPrivateKeyDisplayTs extends Vue {
...
+   * It is necessary to observe the CurrentWallet, 
+   * so that we can access to init. 
+   */
+  @Watch('wallet')
+  observeCurrentWallet(){
+    this.init()
+  }
+  public destroyed(){
+    this.init()
+  }
```
Comment (evias) [inline src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts:141]: semantically, this doesn't make sense. Please rename `init` to `reset()`

File: src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts [hunk 3]
```diff
@@ -90,35 +90,25 @@ export class ProtectedPrivateKeyDisplayTs extends Vue {
   }
   /// end-region computed properties getter/setter
 
+  public init(){
+    this.hasPlainPrivateKey = false
+    this.secondsCounter = defaultCount
+    this.countInterval && clearInterval(this.countInterval)
+    this.countInterval = null
```
Comment (evias) [inline src/components/ProtectedPrivateKeyDisplay/ProtectedPrivateKeyDisplayTs.ts:97]: Please, also update `plainInformation` property to a null'd value.

**Comment by vincent-lee90** [comment]
changed as the comments above.

**Pull request merged**